tests/km_core_tests.bats

```diff
@@ -1819,28 +1820,35 @@ EOF
       tries=$(($tries - 1))
    done
    assert [ $tries -ne 0 ]
-   rm -f $WORKDIR/waiting
+   run rm -f $WORKDIR/waiting
+   assert_success
```

Comment:
Isn't `rm -f` always successful? Unless `waiting` might somehow become a directory.

I might misunderstand something here, but it seems to me there is no need to check this, nor to do `run rm`

Reply:
The run is not needed, nor the assert_success